core/src/main/java/com/novoda/noplayer/internal/exoplayer/ExoPlayerTwoImpl.java

```diff
@@ -178,9 +186,22 @@ public void onSurfaceHolderReady(SurfaceHolder surfaceHolder) {
                 exoPlayer.loadVideo(surfaceHolder, drmSessionCreator, uri, contentType, forwarder, mediaCodecSelector);
             }
         };
+
+        assertPlayerViewIsAttached();
+        forceSurfaceToBeCreated();
         surfaceHolderRequester.requestSurfaceHolder(onSurfaceReadyCallback);
     }
 
+    private void assertPlayerViewIsAttached() {
+        if (playerView == null) {
+            throw new IllegalStateException("A PlayerView must be attached in order to loadVideo");
```

Comment:
it's null'd out already in detach. I guess the assertion is added because it was added to the androidImpl